plugins/modules/azure_rm_registrationassignment.py

```diff
@@ -0,0 +1,381 @@
+#!/usr/bin/python
+#
+# Copyright (c) 2020 GuopengLin, (@t-glin)
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = '''
+---
+module: azure_rm_registrationassignment
+version_added: '1.3.0'
+short_description: Manage Azure RegistrationAssignment instance
+description:
+    - Create, update and delete instance of Azure RegistrationAssignment.
+options:
+    scope:
+        description:
+            - Scope of the resource.
+        required: true
+        type: str
+    registration_assignment_id:
+        description:
+            - Guid of the registration assignment.
+        type: str
+    expand_registration_definition:
```

Comment:
return the registration definition id is enough

Reply:
This is defined in the API, return detail info or not!